Description: Completely disable the usage of pread/pwrite
 .
 This is an old patch and should be re-checked.
Origin: vendor
Bug-Debian: http://bugs.debian.org/261311
Forwarded: no
Last-Update: 2010-01-18

--- pkg-php.orig/acinclude.m4
+++ pkg-php/acinclude.m4
@@ -1219,7 +1219,7 @@ $1
     }
 
   ],[
-    ac_cv_pwrite=yes
+    ac_cv_pwrite=no
   ],[
     ac_cv_pwrite=no
   ],[
@@ -1248,7 +1248,7 @@ $1
     exit(0);
     }
   ],[
-    ac_cv_pread=yes
+    ac_cv_pread=no
   ],[
     ac_cv_pread=no
   ],[
